### DISCUSSION TITLE
newly created module not visible, only resetdb works

### DISCUSSION THREAD

**Discussion by weinni2000** [Tecnativa/doodba#633] [Q&A]
Describe the bug
I'm new to dooba so maybe it's just a wrong usage.
But if I manually add a new module in one of the repos it's not visible in the frontend.
Even if I rebuild the container with "invoke img-build" it's still not visible. Only if I do a "invoke resetdb" it's accessable. What do I do wrong, or where can be the issue?

To Reproduce
Affected versions:
18

Add a new module in one of the repos
Try to install the new module
Keep the current state of the db but add the new module.

The addons.yaml looks like this and an invoke git-aggregate also was made before.

mytime:

"*"

**Comment by pedrobaeza** [comment]
Describe what "I manually add a new module" means exactly with all the steps you did.

**Reply by weinni2000** [reply]
creating a new folder (or copying an existing template) with minimal setup (manifest one model) after changing the basic data (name, etc) I expect that I can install the module (after refreshing the module list in the frontend). In my case this was not possible only after the resetdb (without changing anything in the module) it shows up.

**Reply by pedrobaeza** [reply]
You are still omitting a lot of information. You say "creating a new folder", but not where. I suppose you are doing that on the `private` folder. Anyway, have you restarted your Odoo instance?

**Comment by weinni2000** [comment]
I have a custom repo which is added in the repo.yaml. and made accessable in addons.yaml  
mytime   
- "*"

![Bildschirmfoto 2024-10-23 um 09 42 57](https://github.com/user-attachments/assets/9dc6e48a-267d-4b67-b5b9-04f8187fbaf0)
It's my understanding that's this is the proper way to do it. 

I restarted it with "invoke start" but I also rebuild the whole container with "invoke img-build" and also run git-aggregate also with invoke.

**Reply by pedrobaeza** [reply]
OK, on that case, you need to restart the Odoo instance: `docker compose restart odoo`, then go to the applications list, and update it.
